utils/misc.ml

```diff
@@ -730,3 +730,16 @@ module MakeHooks(M: sig
   let apply_hooks sourcefile intf =
     fold_hooks !hooks sourcefile intf
 end
+
+(* showing configuration and configuration variables *)
+let show_config_and_exit () =
+  Config.print_config stdout;
+  exit 0
+
+let show_config_variable_and_exit var =
+  match Config.config_var var with
+  | Some valu ->
+      print_endline valu;
```

Comment:
Why do this - **not** doing this will be amazingly wonderful for compatibility between Windows/Cygwin and the rest of the world where shell scripting is concerned (no more `tr -d '\r'` all over the place)

Reply:
You mean why include a line break at the end instead of using `print_string`?

Reply:
Yes, indeed - sorry for being somewhat less than clear!